sample-apps/bundles-price-per-component/shopify.app.toml

```diff
@@ -0,0 +1,4 @@
+# This file stores configurations for your Shopify app.
+# It must exist at the root of your app.
+
+scopes = ""
```

Comment:
If it's an extension only app, then probably not. But, if we did/should have the app alongside it to register the cart transform (as you mention [here](https://github.com/Shopify/function-examples/pull/407#issuecomment-1863067148)), then we should. I do agree it might be worthwhile to have it, otherwise there would be the confusion as to why the cart transform function wouldn't do anything after the app is installed.
@Stefan, what do you think?